Lib/_pyrepl/console.py

```diff
@@ -19,8 +19,14 @@
 
 from __future__ import annotations
 
+import sys
+
 from abc import ABC, abstractmethod
 from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import IO
```

Comment:
I think we probably want to do this instead of `if TYPE_CHECKING`, so that we can avoid importing `typing` altogether at runtime (thereby speeding up the startup time for the REPL).

```suggestion

TYPE_CHECKING = False

if False:
    from typing import IO
```

We do similar things here:

https://github.com/python/cpython/blob/70b07aa4153c1a914a3d69307d5b258cf7ed16ab/Lib/_pyrepl/unix_console.py#L43-L49